src/network/netdev/macvlan.c

```diff
@@ -52,6 +54,13 @@ static int netdev_macvlan_fill_message_create(NetDev *netdev, Link *link, sd_net
                         return log_netdev_error_errno(netdev, r, "Could not append IFLA_MACVLAN_MODE attribute: %m");
         }
 
+        // set the nopromisc flag if Promiscuous= of the link is explicitly set to false
```

Comment:
we use // comments as "fixme" style comments, and /* */ comments as explanatory comments. this is the latter, hence please use /**/.

See https://systemd.io/CODING_STYLE